recipes/julia-conda-packaging/scripts/activate.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+clones_dir="${CONDA_PREFIX}/share/julia/conda_clones"
```

Comment:
Can you make this inline in line 7? We don't want to set variables that we don't want the user to see.

Reply:
Done in db247a0caddd5282b18af3331af4c3763d99f4c9